presto-main/src/main/java/com/facebook/presto/operator/NestedLoopJoinOperator.java

```diff
@@ -41,10 +41,10 @@
     {
         private final int operatorId;
         private final PlanNodeId planNodeId;
-        private final JoinBridgeManager<NestedLoopJoinPagesBridge> joinBridgeManager;
+        private final JoinBridgeManager<NestedLoopJoinBridge> joinBridgeManager;
         private boolean closed;
 
-        public NestedLoopJoinOperatorFactory(int operatorId, PlanNodeId planNodeId, JoinBridgeManager<NestedLoopJoinPagesBridge> nestedLoopJoinPagesSupplierManager)
+        public NestedLoopJoinOperatorFactory(int operatorId, PlanNodeId planNodeId, JoinBridgeManager<NestedLoopJoinBridge> nestedLoopJoinPagesSupplierManager)
```

Comment:
Do you want to rename the parameter here?  I don't care either way, jut wanted to point it out.